sockeye/decoder.py

```diff
@@ -148,6 +152,22 @@ def __init__(self,
                                                                      prefix="final_process_",
                                                                      num_hidden=self.config.model_size)
 
+    def state_structure(self):
```

Comment:
Could you add a return type annotation?
```suggestion
    def state_structure(self) -> str:
```